src/models/subscribeDao.js

```diff
@@ -3,15 +3,31 @@ const { AppDataSource } = require("./dataSource");
 const createSubscribe = async (startDate, endDate) => {
   return await AppDataSource.query(
     `
-    INSERT INTO subscribes (
-      start_date,
-      end_date
-    ) VALUES (?, ?);
+      INSERT INTO subscribes (start_date, end_date)
+      VALUES (?, ?);
     `,
     [startDate, endDate]
   );
 };
 
-module.exports = {
-  createSubscribe,
+const userSubscriprionCheck = async (id) => {
+  const query = await AppDataSource.query(
+    `
+    SELECT subscribes.end_date
+    FROM subscribes
+    JOIN users ON subscribes.id = users.id
+    WHERE users.id = ?
+  `,
+    [id]
+  );
+  return query;
```

Comment:
```suggestion
const getSubscribeInfoByUserId = async (id) => {
  const query = await AppDataSource.query(
    `
    SELECT subscribes.end_date
    FROM subscribes
    JOIN users ON subscribes.id = users.id
    WHERE users.id = ?
  `,
    [id]
  );
  return query;
```

1. 함수 이름에 오타 나지 않게 스펠링 체크 잘 해주세요.
2. dao단에서 생성하는 함수의 이름은 create, read(get), update, delete 로 시작하는 것이 일반적입니다. 그리고 어떤 Entity를 조작하는지, (subscribeInfo)를 명시해 주세요.
3. SELECT할 때 필터링하는 조건이 외부 다른 entity의 id라면 함수명에 명시해 주시는 것이 좋습니다(userId). 그렇지 않고 그냥 id를 인자로 받으면 자연스럽게 subscribe의 id로 다른 개발자에게 오인되기 쉽습니다 :)

Reply:
유의하겠습니다! 함수명 변경했습니다